quarkus-next: java.util.NoSuchElementException: No value present causes quarkus-server build failure

### Description

`kc.log-file` property doesn't have to be present during the build. This can potentially cause an exception in `LoggingPropertyMappers`. Specifically, `resolveFileLogLocation` transformer method can call `.endsWith()` over non-existent/null value.

This problem was spotted in the `quarkus-next` branch. I also came up with a fix that stabilized the `quarkus-next` build phase while testing it locally. I'll sent a PR in a moment.

Here is the stack trace:
```
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:build (default) on project keycloak-quarkus-server-app: Failed to build quarkus application: No value present -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:build (default) on project keycloak-quarkus-server-app: Failed to build quarkus application
...
Caused by: java.util.NoSuchElementException: No value present
    at java.util.Optional.get (Optional.java:143)
    at org.keycloak.quarkus.runtime.configuration.mappers.LoggingPropertyMappers.resolveFileLogLocation (LoggingPropertyMappers.java:211)
    at org.keycloak.quarkus.runtime.configuration.mappers.PropertyMapper.transformValue (PropertyMapper.java:254)
    at org.keycloak.quarkus.runtime.configuration.mappers.PropertyMapper.getConfigValue (PropertyMapper.java:152)
    at org.keycloak.quarkus.runtime.configuration.mappers.PropertyMappers.getValue (PropertyMappers.java:69)
    at org.keycloak.quarkus.runtime.configuration.PropertyMappingInterceptor.getValue (PropertyMappingInterceptor.java:85)
    at io.smallrye.config.SmallRyeConfigSourceInterceptorContext.proceed (SmallRyeConfigSourceInterceptorContext.java:25)
    at io.smallrye.config.SmallRyeConfig.getConfigValue (SmallRyeConfig.java:382)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadOperation.getDefaults (BuildTimeConfigurationReader.java:1219)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadOperation.getDefaults (BuildTimeConfigurationReader.java:1233)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadOperation.getDefaults (BuildTimeConfigurationReader.java:1233)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadOperation.getDefaults (BuildTimeConfigurationReader.java:1233)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadOperation.getDefaults (BuildTimeConfigurationReader.java:1233)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadOperation.getDefaults (BuildTimeConfigurationReader.java:1201)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadOperation.run (BuildTimeConfigurationReader.java:525)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader.lambda$readConfiguration$0 (BuildTimeConfigurationReader.java:424)
    at io.smallrye.config.SecretKeys.doUnlocked (SecretKeys.java:28)
    at io.quarkus.deployment.configuration.BuildTimeConfigurationReader.readConfiguration (BuildTimeConfigurationReader.java:424)
    at io.quarkus.deployment.ExtensionLoader.loadStepsFrom (ExtensionLoader.java:140)
    at io.quarkus.deployment.QuarkusAugmentor.run (QuarkusAugmentor.java:107)
    at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment (AugmentActionImpl.java:333)
    at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication (AugmentActionImpl.java:175)
    at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:133)
    at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:169)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    ...
[ERROR]
```

--- a/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java
+++ b/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java
@@ -208,13 +208,7 @@ public final class LoggingPropertyMappers {
     }
 
     private static Optional<String> resolveFileLogLocation(Optional<String> value, ConfigSourceInterceptorContext configSourceInterceptorContext) {
-        String location = value.get();
-
-        if (location.endsWith(File.separator)) {
-            return of(location + LoggingOptions.DEFAULT_LOG_FILENAME);
-        }
-
-        return value;
+        return value.map(location -> location.endsWith(File.separator) ? location + LoggingOptions.DEFAULT_LOG_FILENAME : location);
     }
 
     private static Level toLevel(String categoryLevel) throws IllegalArgumentException {

